atak-civ/plugin-examples/plugintemplate/app/src/main/java/com/atakmap/android/plugintemplate/PluginTemplateDropDownReceiver.java

```diff
@@ -37,6 +45,7 @@
     private RecyclerView mainScreenTimerList;
     private static ArrayList<ActiveTimer> timers = new ArrayList<>();
     private TimerListAdapter adapter;
+    public static ArrayList<Timer> presets;
```

Comment:
Is there a reason we storing the preset list in the home screen drop down, I feel like it might be better if it were in the preset screen drop down?

Reply:
I need to read it upon loading the app so we have an idea of what the existing presets are before overwriting the file. It will be more clear why we do this when I walk everyone through it on Monday